tests/mytest.py

```diff
@@ -0,0 +1,98 @@
+import time
+
+from slogpy.slog import Slog as slog
+from slogpy.progress import (SlogProgress, get_progress,
+    get_progress_counting, get_progress_counting_with_time)
+from slogpy.slog import Slog as slog
+import click
+slog.initialize(module='widget') # optional, but highly recommended
+
+slog.info('Log something at the info level')
+slog.debug('log some debugging...this will only go to file unless you set the logging level')
+slog.annoy('You need to add handling for this!')
+slog.warn('hey, this is a warning')
+slog.error('Something bad happened')
+slog.fatal('Oh no, Mr. Bill! Something REALLY bad happened')
+
+# if you want to tell the user where the log is (after any logging!)
+print(f'Log written to: {slog.get_logging_path()}')
+
+# You can call SlogProgress() just like you would rich.progress.Progress
+# or you can use one of the predefined factory methods. If you have a
+# progress bar that you use often, add a factory for it in
+# slogpy/slogpy/progress.py
+
+def test_progress():
+    my_iterable = ['apple', 'pear', 'orange', 'banana', 'kiwi', 'plum', 'durian', 'strawberry', 'grape', 'grapefruit']
+
+    progress = SlogProgress()
+    with progress:
+        for fruit in progress.track(my_iterable, description='pick fruit'):
+            slog.info(f'working on {fruit}')
+            time.sleep(0.5)
+
+    progress = get_progress_counting()
+    with progress:
+        for fruit in progress.track(my_iterable, description='pick fruit'):
+            slog.info(f'working on {fruit}')
+            time.sleep(0.5)
+
+    progress = get_progress_counting_with_time()
+    with progress:
+        for tick in progress.track(range(0,10093), description='pick fruit'):
+            if tick % 427 == 0:
+                slog.info(f'{tick} mod 7 is 0')
+            time.sleep(0.0003)
+
+
+if __name__ == '__main__':
+    test_progress()
+
+def some_function(name, title, number=27):
```

Comment:
This isn't a test, should probably be removed.  These files should have unit tests only, methods such as these to show sample usage are helpful but belong elsewhere in an examples directory or project wiki